### PR TITLE
Translate intro to storybook french (vuejs)

### PR DESCRIPTION
**Pull request by jeanphibaconnais**
Hi, 
i've just finished to translate the first part (intro to Storybook) for vuejs in french

### CODE REVIEW

File: content/intro-to-storybook/vue/fr/conclusion.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<br/>
+[🌎 **Instance Storybook déployée**](https://clever-banach-415c03.netlify.com/)
+
+Storybook est un puissant outil pour React, React Native, Vue, Angular, Svelte et beaucoup d'autres frameworks. Il y a une communauté de développeurs prospère et une nombre important d'addons. Cette introduction égratigne la surface de ce qui est possible. Nous sommes convaincus qu'une fois que vous aurez adopté Storybook, vous serez impressionnés par la manière de produire des interfaces utilisateurs durables.
+
+## En apprendre plus
+
+Vous voulez en apprendre plus ? Voici des ressources utiles :
+
+- [**Documentation Officielle de Storybook**](https://storybook.js.org/basics/introduction/) possède une documentation de l'API, des liens de la communauté et une galerie d'addons.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/fr/conclusion.md:18]: @jeanphibaconnais when you're able, can you please adjust the link for "Documentation Officielle de Storybook" to point at https://storybook.js.org/docs/vue/get-started/introduction?

File: content/intro-to-storybook/vue/fr/conclusion.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+- [**The Delightful Storybook Workflow**](https://blog.hichroma.com/the-delightful-storybook-workflow-b322b76fd07) met en évidence les meilleures pratiques de workflow utilisées par les équipes de haute vélocité de Squarespace, Major League Soccer, Discovery Network, et Apollo GraphQL.
+
+- [**Visual Testing Handbook**](https://www.chromatic.com/book/visual-testing-handbook) plonge profondément dans l'utilisation de Storybook pour visualiser les composants de test. Livre numérique gratui de 31 pages.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/fr/conclusion.md:22]: @jeanphibaconnais when you're able can you please adjust the link for "Visual Testing Handbook" to point at https://www.learnstorybook.com/visual-testing-handbook/ ?

File: content/intro-to-storybook/vue/fr/conclusion.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+title: 'Conclusion'
+description: 'Mettez toutes vos connaissances ensemble et apprenez plus de techniques Storybook'
+---
+
+Félicitations ! Vous avez créé votre première interface utilisateur avec Storybook. En cours de route, vous avez appris comment construire, composer, tester et déployer des composants graphiques. Si vous avez suivi, votre projet et instance Storybook déployée devraient ressembler à ceci :
+
+[📕 **Projet GitHub : chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code/tree/vue)
+<br/>
+[🌎 **Instance Storybook déployée**](https://clever-banach-415c03.netlify.com/)
+
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/fr/conclusion.md:11]: @jeanphibaconnais when you're able, can you please update the link to point at https://clever-banach-415c03.netlify.app/ ?

File: content/intro-to-storybook/vue/fr/deploy.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+Configurez Netlify en surlignant la commande de build à construire dans son CI et le répertoire dans lequel le site statique est généré. Pour la branche, choisissez `master`. Le répertoire est `storybook-static`. La commande de build utilise `yarn build-storybook`.
+
+![Paramètres Netlify](/intro-to-storybook/netlify-settings.png)
+
+<div class="aside"><p>Si votre déploiement échoue avec Netlify, ajoutez l'option <a href="https://storybook.js.org/docs/configurations/cli-options/#for-build-storybook">--quiet </a>  à votre script <code>build-storybook</code>.</p></div>
+
+Soumettez le formulaire pour construire et déployer le code de la branche `master` du repo taskbox.
+
+Une fois terminé, nous verrons un message de confirmation sur Netlify avec un lien vers le Storybook taskbox en ligne. Si vous suivez le tutoriel, votre Storybook déployé devrait être en ligne [comme cela](https://clever-banach-415c03.netlify.com/).
+
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/fr/deploy.md:94]: @jeanphibaconnais when you're able can you adjust the link "come cela" to point at  https://clever-banach-415c03.netlify.app/ ?

File: content/intro-to-storybook/vue/fr/conclusion.md (hunk 1 above)
Comment (jeanphibaconnais) [reply content/intro-to-storybook/vue/fr/conclusion.md]: Hi @jonniebigodes , i've just correct the link.

File: content/intro-to-storybook/vue/fr/conclusion.md (hunk 2 above)
Comment (jeanphibaconnais) [reply content/intro-to-storybook/vue/fr/conclusion.md]: it's ok for this one.

File: content/intro-to-storybook/vue/fr/conclusion.md (hunk 3 above)
Comment (jeanphibaconnais) [reply content/intro-to-storybook/vue/fr/conclusion.md:11]: it's ok for this one.

File: content/intro-to-storybook/vue/fr/deploy.md (hunk 4 above)
Comment (jeanphibaconnais) [reply content/intro-to-storybook/vue/fr/deploy.md]: it's ok for this one.

File: content/intro-to-storybook/vue/fr/conclusion.md (hunk 1 above)
Comment (jonniebigodes) [reply content/intro-to-storybook/vue/fr/conclusion.md]: Thank you for updating it.

File: content/intro-to-storybook/vue/fr/conclusion.md (hunk 2 above)
Comment (jonniebigodes) [reply content/intro-to-storybook/vue/fr/conclusion.md]: Once again thank you for making the change

File: content/intro-to-storybook/vue/fr/deploy.md (hunk 4 above)
Comment (jonniebigodes) [reply content/intro-to-storybook/vue/fr/deploy.md]: Thanks once more for updating it

File: content/intro-to-storybook/vue/fr/screen.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+Nous avons commencé par le composant le plus bas avec `Task`, puis nous sommes passés à `TaskList`, maintenant nous sommes ici avec une interface utilisateur à écran complet. Notre `InboxScreen` contient un composant de conteneur imbriqué et comprend des histoires d'accompagnement.
+
+<video autoPlay muted playsInline loop style="width:480px; height:auto; margin: 0 auto;">
+  <source
+    src="/intro-to-storybook/component-driven-development-optimized.mp4"
+    type="video/mp4"
+  />
+</video>
+
+[**Le développement basé sur les composants (Component-Driven Development)**](https://blog.hichroma.com/component-driven-development-ce1109d56c8e) vous permet d'augmenter progressivement la complexité à mesure que vous montez dans la hiérarchie des composants. Parmi les avantages figurent un processus de développement plus ciblé et une couverture accrue de toutes les permutations possibles de l'interface utilisateur. En bref, CDD vous aide à créer des interfaces utilisateur de meilleure qualité et plus complexes.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/fr/screen.md:212]: @jeanphibaconnais sorry for not addressing this earlier, but when you're able, please adjust the link "Le développement basé sur les composants (Component-Driven Development)" to point at https://www.componentdriven.org/

File: content/intro-to-storybook/vue/fr/simple-component.md [hunk 6]
```diff
@@ -0,0 +1,281 @@
+---
+title: 'Construire un simple composant'
+tocTitle: 'Un simple composant'
+description: 'Construire un simple composant en isolation'
+commit: b2274bd
+---
+
+Nous allons construire notre propre interface utilisateur en suivant la méthode [Component-Driven Development](https://blog.hichroma.com/component-driven-development-ce1109d56c8e). C'est un processus qui créé des interfaces utilisateurs du début à la fin, en commençant par les composants et terminant avec les écrans. CDD vous aide à adapter la complexité à laquelle vous êtes confronté lors de la création de l'interface utilisateur.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/fr/simple-component.md:8]: @jeanphibaconnais when you're able please adjust the link "Component-Driven Development" to point at https://www.componentdriven.org/

File: content/intro-to-storybook/vue/fr/using-addons.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+// pareil qu'avant
+```
+
+Maintenant, un nouvel onglet "Knobs" devrait apparaître à côté de l'onglet "Action Logger" dans le volet du bas.
+
+Comme documenté [ici](https://github.com/storybooks/storybook/tree/master/addons/knobs#object), l'`objet` know accepte une étiquette et un objet par défaut comme paramètres. L'étiquette est constante et apparaît à gauche d'un champ de texte dans votre panneau de modules complémentaires. L'objet que vous avez transmis sera représenté sous la forme d'un objet blob JSON modifiable. Tant que vous soumettez un JSON valide, votre composant s'ajustera en fonction des données transmises à l'objet!
+
+## Les modules complémentaires font évoluer la portée de votre livre de contes
+
+Non seulement votre instance Storybook sert d'[environnement CDD](https://blog.hichroma.com/component-driven-development-ce1109d56c8e) merveilleux, mais nous fournissons maintenant une source de documentation interactive. Les PropTypes sont excellents, mais un concepteur ou quelqu'un de complètement nouveau dans le code d'un composant sera capable de comprendre son comportement très rapidement via Storybook avec l'addon de boutons implémenté.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/vue/fr/using-addons.md:105]: @jeanphibaconnais when you're able please adjust the link "environnement CDD" to point at https://www.componentdriven.org/

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@jeanphibaconnais sorry for this, but there were some items that needed a bit of attention. When you're able please address the changes. 

I'm currently working on finding someone that can help proofread so that we can get it merged soon.

Stay safe

File: content/intro-to-storybook/vue/fr/using-addons.md (hunk 7 above)
Comment (jeanphibaconnais) [reply content/intro-to-storybook/vue/fr/using-addons.md]: it's modified 😄

File: content/intro-to-storybook/vue/fr/simple-component.md (hunk 6 above)
Comment (jeanphibaconnais) [reply content/intro-to-storybook/vue/fr/simple-component.md]: it's modified 😄

File: content/intro-to-storybook/vue/fr/screen.md (hunk 5 above)
Comment (jeanphibaconnais) [reply content/intro-to-storybook/vue/fr/screen.md]: no problem @jonniebigodes 😄  it's modified

**Comment by Dooxe59** [comment]
Hi :) 

I can help you to validate/translate the remaining pages of the documentation if you wish to do so

**Comment by jonniebigodes** [comment]
@Dooxe59 yes please, we would really appreciate the help with proofreading, so that this can be merged soon.

File: content/intro-to-storybook/vue/fr/conclusion.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
+---
+title: 'Conclusion'
+description: 'Mettez toutes vos connaissances ensemble et apprenez plus de techniques Storybook'
+---
+
+Félicitations ! Vous avez créé votre première interface utilisateur avec Storybook. En cours de route, vous avez appris comment construire, composer, tester et déployer des composants graphiques. Si vous avez suivi, votre projet et instance Storybook déployée devraient ressembler à ceci :
```
Comment (Dooxe59) [inline content/intro-to-storybook/vue/fr/conclusion.md:6]: I suggest a small change for this translation:

Félicitations ! Vous avez créé votre première interface utilisateur avec Storybook. En cours de route, vous avez appris **à** construire, composer, tester et déployer des composants graphiques. Si vous avez bien suivi, votre projet et **votre** instance Storybook **une fois** déployée devraient ressembler à ceci :

File: content/intro-to-storybook/vue/fr/conclusion.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+[📕 **Projet GitHub : chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code/tree/vue)
+<br/>
+[🌎 **Instance Storybook déployée**](https://clever-banach-415c03.netlify.com/)
+
+Storybook est un puissant outil pour React, React Native, Vue, Angular, Svelte et beaucoup d'autres frameworks. Il y a une communauté de développeurs prospère et une nombre important d'addons. Cette introduction égratigne la surface de ce qui est possible. Nous sommes convaincus qu'une fois que vous aurez adopté Storybook, vous serez impressionnés par la manière de produire des interfaces utilisateurs durables.
```
Comment (Dooxe59) [inline content/intro-to-storybook/vue/fr/conclusion.md:12]: Reverse words (puissant outil): 

Storybook est un outil puissant...

Comment (Dooxe59) [reply content/intro-to-storybook/vue/fr/conclusion.md:12]: Proposition:

Storybook est un outil puissant pour React, React Native, Vue, Angular, Svelte et beaucoup d'autres frameworks. Il possède une importante communauté de développeurs et un nombre important d'extensions. Cette introduction survole la surface de ce qui est possible de faire. Nous sommes convaincus qu'une fois que vous aurez adopté Storybook, vous serez impressionnés par la manière de produire des interfaces utilisateurs durables.

File: content/intro-to-storybook/vue/fr/contribute.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
+---
+title: 'Contribuer'
+description: 'Aider à partager Storybook dans le monde'
+---
+
+Les contributions à Learn Storybook sont encouragées ! S'il s'agit de petites corrections comme de la grammaire our de la ponctuation, ouvrez une pull request. S'il s'agit d'importants changements, [ajouter une issue](https://github.com/chromaui/learnstorybook.com/issues) pour échanger.
```
Comment (Dooxe59) [inline content/intro-to-storybook/vue/fr/contribute.md:6]: our not exist in french :)
It must be 'ou'

File: content/intro-to-storybook/vue/fr/contribute.md [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+
+Learn Storybook a été principalement créé et maintenu par la communauté, nous avons donc besoin de l'aide de tous pour le garder à jour et nous assurer que <TODO>and ensure any rough edges are smoothed out over time.</TODO> Toute aide est appréciée !
```
Comment (Dooxe59) [inline content/intro-to-storybook/vue/fr/contribute.md:8]: Translation not finished here :)

Learn Storybook a été principalement créé et maintenu par la communauté, nous avons donc besoin de l'aide de tous pour le garder à jour et nous assurer que ...
les imperfections seront corrigés avec le temps.

**Review by Dooxe59 — COMMENTED** [review]
> @Dooxe59 yes please, we would really appreciate the help with proofreading, so that this can be merged soon.

What is the procedure for making my returns? 

I started by commenting the conclusion.md file, contribute.md but I'm not sure this is the right way to go?


I am available and motivated to help you translate all this ;)

File: content/intro-to-storybook/vue/fr/contribute.md (hunk 10 above)
Comment (jeanphibaconnais) [reply content/intro-to-storybook/vue/fr/contribute.md:6]: Thanks for all yours comments, i will change those next week !

File: content/intro-to-storybook/vue/fr/conclusion.md (hunk 8 above)
Comment (jeanphibaconnais) [reply content/intro-to-storybook/vue/fr/conclusion.md:6]: Ok effectivement, j'ajoute ça de suite

File: content/intro-to-storybook/vue/fr/contribute.md (hunk 11 above)
Comment (jeanphibaconnais) [reply content/intro-to-storybook/vue/fr/contribute.md]: outch, sorry for this ! it's changed

File: content/intro-to-storybook/vue/fr/conclusion.md (hunk 9 above)
Comment (jeanphibaconnais) [reply content/intro-to-storybook/vue/fr/conclusion.md]: it's changed. Thanks for all yours comments.

File: content/intro-to-storybook/vue/fr/contribute.md (hunk 10 above)
Comment (jeanphibaconnais) [reply content/intro-to-storybook/vue/fr/contribute.md]: it's changed :)

**Review by Dooxe59 — APPROVED** [review]

**Comment by jonniebigodes** [comment]
@jeanphibaconnais sorry for not addressing this sooner. Is this good to go? Can we merge it?

Stay safe

**Comment by jeanphibaconnais** [comment]
> @jeanphibaconnais sorry for not addressing this sooner. Is this good to go? Can we merge it?
> 
> Stay safe

Hi @jonniebigodes , yes it's ok for me, all the comments are resolved.

**Comment by jonniebigodes** [comment]
Thank you both for the time and effort you put into this. We really appreciate the translation. @jeanphibaconnais the Vue tutorial is going to be updated to Storybook 6.0. If you have the time, we would love for you to do the honors and update the French translation. Following up on this pull request. Let me know and we'll move from there.

Sounds good?

**Comment by jeanphibaconnais** [comment]
Hi @jonniebigodes  yes of course. I will try to look that in the beginning of decembre

**Pull request merged**